source/_verkenning.Rmd

```diff
@@ -0,0 +1,424 @@
+```
+
+```{r, fig.cap = "Rarefaction curve van het aantal soorten {{group}} in functie van aantal individuen gevangen met pan traps."}
+rarefaction_result_2 %>%
+  ggplot(
+    aes(
+      x = individuals, y = richness)
+    ) +
+  geom_line(
+    aes(
+      colour = location_code)) +
+  geom_ribbon(
+    aes(
+      ymin = lwr, ymax = upr, fill = location_code), alpha = 0.2) +
+  xlab("Rarefied number of individuals") +
+  ylab("Number of (morpho)species") +
+  facet_grid(paste("maand:", month) ~ method_combi, scales = "free")
+```
+
+
+
+#### Saturatie in aantal deeltransecten
+
+```{r}
+rarefaction_result_3 <- {{group}} %>%
+  filter(method_cd == "TS") %>%
+  complete(
+    nesting(
+      location_code, method_cd, spring_code, level, month,
+      sampling_site_cd),
+    species_nm,
+    fill = list(no_ind = 0)) %>%
+  group_by(location_code, method_cd, spring_code,
+           sampling_site_cd, species_nm) %>%
+  summarise(n = sum(no_ind)) %>%
+  filter(sum(n) > 0) %>%
+  group_by(location_code, method_cd, spring_code) %>%
+  nest() %>%
+  mutate(rarefaction = map(data, ~ rarefaction(.x))) %>%
+  select(-data) %>%
+  unnest(rarefaction) %>%
+  ungroup()
+```
+
+
+```{r, fig.cap = "Rarefaction curve van het aantal soorten {{group}} in functie van aantal deeltransecten gelopen."}
+rarefaction_result_3 %>%
+  ggplot(
+    aes(
+      x = sites, y = richness)
+    ) +
+  geom_line(
+    aes(
+      colour = location_code)) +
+  geom_ribbon(
+    aes(
+      ymin = lwr, ymax = upr, fill = location_code), alpha = 0.2) +
+  xlab("Rarefied number of transect 50m segments") +
+  ylab("Number of (morpho)species") +
+  facet_grid(~ spring_code, scales = "free")
+```
+
+```{r, fig.cap = "Rarefaction curve van het aantal soorten {{group}} in functie van aantal individuen gevangen tijdens transect walks."}
+rarefaction_result_3 %>%
+  ggplot(
+    aes(
+      x = individuals, y = richness)
+    ) +
+  geom_line(
+    aes(
+      colour = location_code)) +
+  geom_ribbon(
+    aes(
+      ymin = lwr, ymax = upr, fill = location_code), alpha = 0.2) +
+  xlab("Rarefied number of individuals") +
+  ylab("Number of (morpho)species") +
+  facet_grid(~ spring_code, scales = "free")
+```
+
+#### Cumulatieve soortenaantallen doorheen de maanden
+
+```{r}
+{{group}}_cumulative_month <- {{group}} %>%
+  group_by(
+    location_code, sampling_site_cd, method_cd, spring_code
+  ) %>%
+  arrange(month, .by_group = TRUE) %>%
+  group_by(
+    location_code, sampling_site_cd, method_cd, spring_code,
+    level, uv, method_combi, month
+  ) %>%
+  summarise(
+    species_list = list(species_nm[!is.na(species_nm)])
+  ) %>%
+  mutate(
+    accumulated_species_list = accumulate(
+      species_list, union, .simplify = FALSE),
+    cumulative_n_species = map_dbl(accumulated_species_list, length),
+    extra_n_species = cumulative_n_species - dplyr::lag(cumulative_n_species),
+    pt_ts = str_sub(sampling_site_cd, 12, 13),
+    month = as.numeric(month)
```

Comment:
Vermoedelijk is month een factor want in de figuur staat nu 1 2 3 4 5 in de x-as. Kan je wellicht oplossen door as.numeric(as.character(month))